addons/delivery/models/stock_picking.py

```diff
@@ -119,6 +119,8 @@ def do_transfer(self):
 
         return res
 
+    do_transfer = action_done # @SLE change me as in stock > picking
```

Comment:
Ok for me if it is ok for @sle-odoo / @pimodoo , I know they have changed the flow for pickings in v11